docs/Makefile

```diff
@@ -75,4 +75,10 @@ multiversion: setup
 
 .PHONY: multiversionpreview
 multiversionpreview: multiversion
-	$(POETRY) run python3 -m http.server 5500 --directory $(BUILDDIR)/dirhtml 
+	$(POETRY) run python3 -m http.server 5500 --directory $(BUILDDIR)/dirhtml
+
+.PHONY: test
+test: setup
+	$(SPHINXBUILD) -b dirhtml $(TESTSPHINXOPTS) $(BUILDDIR)/dirhtml
+	@echo
+	@echo "Build finished. The HTML pages are in $(BUILDDIR)/dirhtml."
```

Comment:
@lauranovich After the line 12 (Makefile), please add the following new line:

```
TESTSPHINXOPTS  = $(ALLSPHINXOPTS) -W --keep-going
```

I'm updating the upgrade guide to reflect this change.

Reply:
@lauranovich Could you please add the new line after the line 13 (Under "# Internal variables.", after "ALLSPHINXOPTS ...")? It's just a aesthetic change to have a Makefile more organized. Now, ``TESTSPHINXOPTS  = $(ALLSPHINXOPTS) -W --keep-going`` is at the end of the file.

Reply:
done